src/bpmailsend.c

```diff
@@ -51,30 +53,57 @@ static int bpmailsend(void) {
      */
     if (content_size > UINT_MAX) {
         (void)fprintf(stderr, "content too large to send\n");
+        free(content);
+        return EXIT_FAILURE;
+    }
+
+    // Compress content using zlib
+    const char magic[] = "ZLIB";  // magic header to identify compressed content
+    uLongf compressed_size = compressBound((uLongf)content_size);
+    Bytef *compressed = malloc(strlen(magic) + compressed_size);
```

Comment:
```suggestion
    /* Compress content using zlib */
    const char magic[] = "ZLIB";  / *magic header to identify compressed content */
    uLongf compressed_size = (uLongf)compressBound((uLong)content_size);
    Bytef *compressed = malloc(strlen(magic) + compressed_size);
```
My suggested changes for line 62 aren't necessary, but I'd prefer if it matches with the API more (uses `uLong` rather than `uLongf` as its argument and returns `uLong`) https://github.com/madler/zlib/blob/5a82f71ed1dfc0bec044d9702463dbdf84ea3b71/zlib.h#L1282